hazelcast/src/main/java/com/hazelcast/partition/impl/InternalPartitionServiceImpl.java

```diff
@@ -718,6 +718,11 @@ public void processPartitionRuntimeState(PartitionRuntimeState partitionState) {
         lock.lock();
         try {
             final Address sender = partitionState.getEndpoint();
+            if (!node.getNodeExtension().isStartCompleted()) {
```

Comment:
Just for my understanding; will this method be called multiple times? Because now we are existing because the start has not completed; but is a new attempt going to be made that eventually can succeed?